Symfony/CS/FixerFactory.php

```diff
@@ -36,6 +36,13 @@
     private $fixers = array();
 
     /**
+     * Fixers by name.
```

Comment:
Would be great to say that it is lazy loaded when getter is used.
To prevent using the variable without getter directly in `FixerFactory`